code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/legionnaire.dm

```diff
@@ -282,11 +301,12 @@
 
 /obj/item/crusher_trophy/legionnaire_spine
 	name = "legionnaire spine"
-	desc = "The spine of a legionnaire.  It almost feels like it's moving..."
+	desc = "The spine of a legionnaire.  With some creativity, you could use it as a crusher trophy.  Alternatively, shaking it might do something as well."
```

Comment:
```suggestion
	desc = "The spine of a legionnaire. With some creativity, you could use it as a crusher trophy. Alternatively, shaking it might do something as well."
```